Unix/http/httpauth.c

```diff
@@ -460,8 +459,12 @@ MI_Boolean Http_EncryptData(_In_ Http_SR_SocketData * handler, _Out_ char **pHea
     gss_buffer_desc output_buffer = { 0 };
     OM_uint32 min_stat, maj_stat;
     int out_flags;
+    unsigned in_flags;
+
+maj_stat = 
+gss_inquire_context (&min_stat, handler->pAuthContext, NULL, NULL, NULL, NULL, &in_flags, NULL, NULL);
 
-    maj_stat = gss_wrap(&min_stat, handler->pAuthContext, handler->negFlags,    // GSS_C_INTEG_FLAG and or GSS_C_PRIV_FLAG
+    maj_stat = gss_wrap(&min_stat, handler->pAuthContext, handler->negFlags| GSS_C_REPLAY_FLAG,    // GSS_C_INTEG_FLAG and or GSS_C_PRIV_FLAG
```

Comment:
Okay, so at 464, you're calling gss_inquire_context, assigning it to maj_stat, and not checking the result. This doesn't seem correct. You do check result for the next call (at 467).